ci-operator/config/openshift/certman-operator/openshift-certman-operator-master.yaml

```diff
@@ -50,6 +50,7 @@ tests:
     from: src
 - as: leading-zero-check
   postsubmit: true
+  presubmit: true
```

Comment:
You cannot add postsubmit and presubmit in the same config

Reply:
I added both as part of the same config due to the response here: https://redhat-internal.slack.com/archives/CBN38N3MW/p1739457362371669?thread_ts=1739395878.023609&cid=CBN38N3MW

Reply:
Add two separate config, one which contains `postsubmit: true` and other without postsubmit and run `make jobs` it will create a postsubmit and presubmit job for you